src/commands/deployment/get.decorator.json

```diff
@@ -47,6 +47,11 @@
       "arg": "-w, --watch",
       "description": "Watch the deployments for a live view",
       "defaultValue": false
+    },
+    {
+      "arg": "--remove-separators",
```

Comment:
to be consistent, can we add an alias?
and we do not add separators, why are we asking user to remove them? :-)
can we have it as `--hide-separators`

Reply:
The default display now includes separators (see [here](https://github.com/microsoft/bedrock/issues/743#issuecomment-548568881)) So we're adding a flag for the user to remove them, if they would like to copy paste the response.

Reply:
i read it as. by default, we are showing the separators. So we are adding a flag for the user to 'hide' them. just my point of view. We are showing and we allowing user to hide them. Or you referring to it as we are adding separators by default and we are adding a flag to remove them.